src/components/Slider/SliderInfix.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { bem, INFIX_KIND } from "./SliderCommons";
+import { useSliderInfix, useSliderSelection } from "./SliderContext";
+import Label from "../Label/Label";
+import Icon from "../Icon/Icon";
+
+const defaultIconProps = {
+  clickable: false,
+  ignoreFocusStyle: true
+};
+
+function getCurrentLabel({ isPostfix, isRange, valueText }) {
+  if (!isRange) {
```

Comment:
maybe will be prettier to set it like this?

if (isRange) return isPostfix ? valueText[1] : valueText[0];
else return valueText;

Reply:
I don't think.
I personally don't like mixes of if's and ternary operator.
I prefer early return than if - else construction.
Of course each approach have cons and pros:
- https://medium.com/swlh/return-early-pattern-3d18a41bba8
- https://betterprogramming.pub/are-early-returns-any-good-eed4b4d03866
But I really think early return much more better :)